ArduPlane/ArduPlane.cpp

```diff
@@ -963,6 +963,28 @@ bool Plane::flight_option_enabled(FlightOptions flight_option) const
     return g2.flight_options & flight_option;
 }
 
+#if AP_SCRIPTING_ENABLED
+// this implements the virtual function defined in AP_Vehicle and already used in Copter
+bool Plane::set_desired_speed(float speed)
```

Comment:
I'm really not sure about using `set_desired_speed` which both copter and rover implement as a ground speed to set airspeed on plane. Firstly because its inconsistent and secondly because it prevents us implementing it as a ground speed in the future (VTOL, taxiing ect)

I would much rather see a new `set_desired_airspeed`.

Reply:
In any case, you would need to update the docs, because the currently say it only work on copter and rover.

https://github.com/ArduPilot/ardupilot/blob/eb8eeba8516f5fc3a83b82a7f03235c2ca6578ae/libraries/AP_Scripting/docs/docs.lua#L2347

Reply:
Glad I asked. I didn't realize that, I'll make the change.

Reply:
I'd like to make set_desired_airspeed() take m/s instead of cm/s. Do you think that makes sense @IamPete1 ?

Reply:
done